src/main/resources/data/computercraft/lua/rom/apis/window.lua

```diff
@@ -235,7 +235,17 @@ function create(parent, nX, nY, nWidth, nHeight, bStartVisible)
     -- @type Window
     -- @see term.Redirect
     local window = {}
-
+    function window.isVisible()
```

Comment:
Can you add a brief documentation comment too please? See `getPosition` of the sort of thing you need to say.